### ISSUE TITLE
Miss assigned label

### ISSUE THREAD

**Issue by Utoooooo** [znxlwm/tensorflow-MNIST-cGAN-cDCGAN#1]
In line 211 of cDCGAN I think the y_label should be randint(0, 10) instead of randint(0, 9), as the function randint does not generate the last digit. This issue also suggested by the poor quality of generate '9', since no such label has actually been passed into the generator.

**Comment by ahmetnarman** [comment]
I encountered with the same issue. When I fix it Iget good results for number 9. Also it makes the code more robust when changing the architecture ie. adding more layers changing hyperparameters etc.
